changelogs/.plugin-cache.yaml

```diff
@@ -0,0 +1,35 @@
+plugins:
```

Comment:
This file should not be checked in (it's best to put it into `.gitignore`).

Reply:
just noticed now, will do it.

> This file should not be checked in (it's best to put it into `.gitignore`).